### PR TITLE
Fix warnings in AdapterExample

### PR DESCRIPTION
**Pull request by cmelchior**
@realm/java

### CODE REVIEW

**Comment by zaki50** [comment]
I got `PrngFix` warning.

please add

```
<issue id="PrngFix" severity="ignore" />
```
to `lint.xml`.

File: examples/adapterExample/lint.xml [hunk 1]
```diff
@@ -0,0 +1,7 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<lint>
+    <issue id="AllowBackup" severity="ignore" />
+    <issue id="GoogleAppIndexingWarning" severity="ignore" />
+    <issue id="IconMissingDensityFolder" severity="ignore" />
+    <issue id="IconLauncherShape" severity="ignore" />
+</lint>
```
Comment (nhachicha) [inline examples/adapterExample/lint.xml]: add new line pls

File: examples/adapterExample/src/main/AndroidManifest.xml [hunk 2]
```diff
@@ -1,9 +1,9 @@
 <?xml version="1.0" encoding="utf-8"?>
-<manifest xmlns:android="http://schemas.android.com/apk/res/android"
-    package="io.realm.examples.realmadapters">
+<manifest xmlns:tools="http://schemas.android.com/tools"
```
Comment (zaki50) [inline examples/adapterExample/src/main/AndroidManifest.xml]: no need to add `xmlns:tools="http://schemas.android.com/tools"`

**Comment by zaki50** [comment]
:+1: other than small comment.

**Comment by dalinaum** [comment]
:+1: besides other comments.

**Pull request merged**